src/librbd/crypto/luks/LoadRequest.cc

```diff
@@ -100,8 +95,23 @@ void LoadRequest<I>::handle_read_header(int r) {
     return;
   }
 
+  const char* type;
```

Comment:
Nit: since this is re-used twice, perhaps a simple helper method in `luks::util` to convert the internal enum to the CRYPT_XYZ constant?

Reply:
The other instance in luks/FormatRequest is a bit different: it also sets the sector size based on the format (luks1/luks2).
So if I switch to using this util in Format, I will double switch on the format (one for converting the format, one for setting the sector size). Then I guess I should assume this conversion utility must return either LUKS1/LUKS2/nullptr?

Reply:
Let's ignore this for now.